docs/source/getting-started-client-mode.md

```diff
@@ -94,5 +102,29 @@ Please see below how a kernel.json would look like for integrating with Spark St
 }
 ```
 
-After making any necessary adjustments such as updating SPARK_HOME or other environment specific configuration, you now should have 
-a new Kernel available which will use Jupyter Enterprise Gateway to execute your notebook cell contents.
\ No newline at end of file
+***Backward compatibility with Jupyter Kernel Gateway***
+
+Jupyter Enterprise Gateway extends Jupyter Kernel Gateway and is 100% compatible with JKG, which enables kernels
+to continue to be run in local client mode by removing the "process_proxy" configuration from the specific kernelspec.
+
+
+```json
+{
+  "language": "python",
+  "display_name": "Spark 2.1 - Python (YARN Client Mode)",
+  "env": {
+    "SPARK_HOME": "/usr/hdp/current/spark2-client",
+    "PYSPARK_PYTHON": "/opt/anaconda3/bin/python",
+    "PYTHONPATH": "${HOME}/.local/lib/python3.6/site-packages:/usr/hdp/current/spark2-client/python:/usr/hdp/current/spark2-client/python/lib/py4j-0.10.4-src.zip",
+    "SPARK_YARN_USER_ENV": "PYTHONUSERBASE=/home/yarn/.local,PYTHONPATH=${HOME}/.local/lib/python3.6/site-packages:/usr/hdp/current/spark2-client/python:/usr/hdp/current/spark2-client/python/lib/py4j-0.10.4-src.zip,PATH=/opt/anaconda2/bin:$PATH",
+    "SPARK_OPTS": "--master spark://127.0.0.1:7077  --name ${KERNEL_ID:-ERROR__NO__KERNEL_ID} --conf spark.yarn.submit.waitAppCompletion=false",
+    "LAUNCH_OPTS": ""
+  },
+  "argv": [
+    "/usr/local/share/jupyter/kernels/spark_2.1_python_yarn_client/bin/run.sh",
+    "{connection_file}",
+    "--RemoteProcessProxy.response-address",
+    "{response_address}"
+  ]
```

Comment:
We should remove the `response-address` entry (and its preceding comma)...
```
,
   "--RemoteProcessProxy.response-address",
   "{response_address}"
```

They probably shouldn't configure the launchers in the run.sh - so this gets a little messy.  Might be best just to say "standard kernelspec configurations" - rather than implying the use of our sample kernelspecs.

Reply:
Good point, I removed the section and just add a new paragraph into the introduction of the section, please check if it looks better now.